app/javascript/src/controller_reference_payment.js

```diff
@@ -0,0 +1,32 @@
+const DefaultController = require('./controller_default');
+const Expander = require('./component_expander');
+
+class ReferencePaymentController extends DefaultController {
+  constructor(app) {
+    super(app);
+
+    switch(app.page.action) {
+      case 'create':
+      case 'index':
+        this.#index();
+      break;
+    }
+  }
+
+  #index() {
+    this.#addExpanderEnhancement();
+  }
+
+  #addExpanderEnhancement() {
+    var $paymentField = $(".fb-payment-field");
+    var $revealedInput = $paymentField.find('[data-component="Expander"]');
+    var $checkbox = $("input[type=checkbox]", $paymentField);
+    new Expander($revealedInput, {
+      activator_source: $checkbox,
+      auto_open: $(".govuk-form-group--error", $paymentField).length,
+      wrap_content: false,
```

Comment:
this may or may not need a bit of tweaking, depending on whether the backend forces the checkbox to be ticked or not. 

when a user is served an error back the expander correctly opens the hidden field, but then when the user goes to click the checkbox it closes the field again and it is not possible to open it after that